lib/Runtime/Base/FunctionBody.h

```diff
@@ -316,13 +316,28 @@ namespace Js
     public:
         // These are public because we don't manage them nor their consistency;
         // the user of this class does.
-        void * address;
+        union address_type_ {
+            address_type_(void *ptr):asPtr(ptr) {}
+            address_type_(Js::JavascriptMethod method):asJsMethod(method) {}
 
-        ProxyEntryPointInfo(void* address, ThreadContext* context = nullptr):
+            void *asPtr;
```

Comment:
> However, it does not apply here.

Maybe I'm missing something, but I see the following code

```
Assert(entryPoint->address.asPtr != nullptr);
return entryPoint->address.asJsMethod;
```

Doesn't it means that both fields of the union are accessed at the same time?